core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/ChangeEvent.java

```diff
@@ -94,7 +94,7 @@ public UpdateDescription getUpdateDescription() {
     return updateDescription;
   }
 
-  public boolean isLocalWritePending() {
+  public boolean hasUncommittedWrites() {
     return localWritePending;
```

Comment:
[nit] should we change the field name as well?

Reply:
sure